UI/WCTDataTreeTabSample/WCTDataTreeTabSample/WCTDataTreeTabSample.Mobile/WCTDataTreeTabSample.Mobile.csproj

```diff
@@ -17,12 +17,15 @@
     <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-android'">21.0</SupportedOSPlatformVersion>
     <SupportedOSPlatformVersion Condition="'$(TargetFramework)'=='net6.0-macos'">10.14</SupportedOSPlatformVersion>
   </PropertyGroup>
-  <ItemGroup>
-	<PackageReference Include="Uno.CommunityToolkit.WinUI.UI.Controls" Version="7.1.100" />	
-	<PackageReference Include="Uno.WinUI" Version="4.5.12" />
-    <PackageReference Include="Uno.WinUI.RemoteControl" Version="4.5.12" Condition="'$(Configuration)'=='Debug'" />
-    <PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.5.12" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
+  <ItemGroup>		
+	<PackageReference Include="CommunityToolkit.WinUI.UI.Controls" Version="7.1.2" />		
```

Comment:
This is not going to work, as it's the WinAppSDK package. You'll need to use the `Uno.` prefixed package

Reply:
Thanks!